packages/cmf/src/onEvent.js

```diff
@@ -5,6 +5,7 @@ import CONSTANT from './constant';
 function serializeEvent(event) {
 	if (event.persist) {
 		event.persist();
+		return {};
```

Comment:
If the event is not used/propagated, why do you still persist it ?

Reply:
I bet that some middleware or js-logger could have some reference on it?

Reply:
No you can t have any reference to it if you don t dispatch it.

Reply:
k thx, updated